_topic_map.yml

```diff
@@ -2684,6 +2684,8 @@ Topics:
   Topics:
   - Name: Preparing your OpenShift cluster for OpenShift Virtualization
     File: preparing-cluster-for-virt
+  - Name: Planning your environment according to Virtualization object maximums
```

Comment:
Made the change. Using {VirtVersion} causes 2.6 to display. This functionality is supported for the OpenShift Virtualization 4.8 release. Should {VirtVersion} be dropped and the version be spelled out as text maybe?

https://deploy-preview-33946--osdocs.netlify.app/openshift-enterprise/latest/virt/install/virt-planning-environment-object-maximums.html

Reply:
We're going to update the value of `{VirtVersion}` to 4.8 as one of our pre-release tasks, so don't worry, it will render correctly when we publish. :)